exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -11,6 +9,8 @@
 - a text variable `cc` with text `hello, world!` including the comma an exclamation
```

Comment:
typo: "comma and exclamation mark"

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/7368/commits/5f05a563a42dfd66953711bb727161fc7e2ae416